var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -58,6 +58,7 @@ class Moab(Package):
     depends_on('mpi')
     depends_on('hdf5+mpi')
     depends_on('netcdf+mpi', when='+netcdf')
+    depends_on('metis')
```

Comment:
> But, when it needs parmetis, it also needs metis.

different from `blas`/`lapack` discussion, here you do not need to add `metis`, It will be brought through `parmetis` dependency. During configure all you do is `--with-parmetis=%s' % spec['parmetis'].prefix`, and `metis` is not used anyhow. So i think this line can go.